### ISSUE TITLE
Crash when adding note on root node

### ISSUE THREAD

**Issue by quentin-bettoum** [phase1geo/Minder#475]
Hello,

I'm using Minder 1.14.0 on Arch Linux and I have a bug.
If I add a note on my root node and I try to select a child node, the program crash.
Then, if I try to open the file again, I get this message (from the terminal):
`Job 1, 'minder Test.minder' terminated by signal SIGSEGV (Address boundary error)`

I managed to reopen my file by editing the XML and removing the root node note.

Here is the XML of a bugged file that cannot be opened anymore.
It works again when I remove `some note` inside the  `<nodenote></nodenote>`.

```
<?xml version="1.0"?>
<minder version="1.14.0" parent-etag="1056773580" etag="1961031014">
  <theme name="dark" label="Dark" index="-1"/>
  <styles>
    <style level="0" isset="true" branchmargin="100" branchradius="25" linktype="curved" linkwidth="4" linkarrow="false" linkdash="solid" nodeborder="rounded" nodewidth="200" nodeborderwidth="4" nodefill="true" nodemargin="10" nodepadding="10" nodefont="Sans 14" nodemarkup="true" connectiondash="dotted" connectionlwidth="1" connectionarrow="fromto" connectionpadding="3" connectionfont="Sans 12" connectiontwidth="100"/>
    <style level="1" isset="true" branchmargin="100" branchradius="25" linktype="curved" linkwidth="4" linkarrow="false" linkdash="solid" nodeborder="rounded" nodewidth="200" nodeborderwidth="4" nodefill="true" nodemargin="8" nodepadding="6" nodefont="Sans 14" nodemarkup="true" connectiondash="dotted" connectionlwidth="1" connectionarrow="fromto" connectionpadding="3" connectionfont="Sans 12" connectiontwidth="100"/>
    <style level="2" isset="true" branchmargin="100" branchradius="25" linktype="curved" linkwidth="4" linkarrow="false" linkdash="solid" nodeborder="rounded" nodewidth="200" nodeborderwidth="4" nodefill="true" nodemargin="8" nodepadding="6" nodefont="Sans 14" nodemarkup="true" connectiondash="dotted" connectionlwidth="1" connectionarrow="fromto" connectionpadding="3" connectionfont="Sans 12" connectiontwidth="100"/>
    <style level="3" isset="true" branchmargin="100" branchradius="25" linktype="curved" linkwidth="4" linkarrow="false" linkdash="solid" nodeborder="rounded" nodewidth="200" nodeborderwidth="4" nodefill="true" nodemargin="8" nodepadding="6" nodefont="Sans 14" nodemarkup="true" connectiondash="dotted" connectionlwidth="1" connectionarrow="fromto" connectionpadding="3" connectionfont="Sans 12" connectiontwidth="100"/>
    <style level="4" isset="true" branchmargin="100" branchradius="25" linktype="curved" linkwidth="4" linkarrow="false" linkdash="solid" nodeborder="rounded" nodewidth="200" nodeborderwidth="4" nodefill="true" nodemargin="8" nodepadding="6" nodefont="Sans 14" nodemarkup="true" connectiondash="dotted" connectionlwidth="1" connectionarrow="fromto" connectionpadding="3" connectionfont="Sans 12" connectiontwidth="100"/>
    <style level="5" isset="true" branchmargin="100" branchradius="25" linktype="curved" linkwidth="4" linkarrow="false" linkdash="solid" nodeborder="rounded" nodewidth="200" nodeborderwidth="4" nodefill="true" nodemargin="8" nodepadding="6" nodefont="Sans 14" nodemarkup="true" connectiondash="dotted" connectionlwidth="1" connectionarrow="fromto" connectionpadding="3" connectionfont="Sans 12" connectiontwidth="100"/>
    <style level="6" isset="true" branchmargin="100" branchradius="25" linktype="curved" linkwidth="4" linkarrow="false" linkdash="solid" nodeborder="rounded" nodewidth="200" nodeborderwidth="4" nodefill="true" nodemargin="8" nodepadding="6" nodefont="Sans 14" nodemarkup="true" connectiondash="dotted" connectionlwidth="1" connectionarrow="fromto" connectionpadding="3" connectionfont="Sans 12" connectiontwidth="100"/>
    <style level="7" isset="true" branchmargin="100" branchradius="25" linktype="curved" linkwidth="4" linkarrow="false" linkdash="solid" nodeborder="rounded" nodewidth="200" nodeborderwidth="4" nodefill="true" nodemargin="8" nodepadding="6" nodefont="Sans 14" nodemarkup="true" connectiondash="dotted" connectionlwidth="1" connectionarrow="fromto" connectionpadding="3" connectionfont="Sans 12" connectiontwidth="100"/>
    <style level="8" isset="true" branchmargin="100" branchradius="25" linktype="curved" linkwidth="4" linkarrow="false" linkdash="solid" nodeborder="rounded" nodewidth="200" nodeborderwidth="4" nodefill="true" nodemargin="8" nodepadding="6" nodefont="Sans 14" nodemarkup="true" connectiondash="dotted" connectionlwidth="1" connectionarrow="fromto" connectionpadding="3" connectionfont="Sans 12" connectiontwidth="100"/>
    <style level="9" isset="true" branchmargin="100" branchradius="25" linktype="curved" linkwidth="4" linkarrow="false" linkdash="solid" nodeborder="rounded" nodewidth="200" nodeborderwidth="4" nodefill="true" nodemargin="8" nodepadding="6" nodefont="Sans 14" nodemarkup="true" connectiondash="dotted" connectionlwidth="1" connectionarrow="fromto" connectionpadding="3" connectionfont="Sans 12" connectiontwidth="100"/>
    <style level="10" isset="true" branchmargin="100" branchradius="25" linktype="curved" linkwidth="4" linkarrow="false" linkdash="solid" nodeborder="rounded" nodewidth="200" nodeborderwidth="4" nodefill="true" nodemargin="8" nodepadding="6" nodefont="Sans 14" nodemarkup="true" connectiondash="dotted" connectionlwidth="1" connectionarrow="fromto" connectionpadding="3" connectionfont="Sans 12" connectiontwidth="100"/>
  </styles>
  <drawarea x="0" y="0" scale="1"/>
  <images/>
  <nodes>
    <node id="0" posx="1011" posy="675.5" width="131" height="54" side="right" fold="false" treesize="54" layout="Horizontal" group="false">
      <style branchmargin="100" branchradius="25" linktype="curved" linkwidth="4" linkarrow="false" linkdash="solid" nodeborder="rounded" nodewidth="200" nodeborderwidth="4" nodefill="true" nodemargin="8" nodepadding="6" nodefont="Sans 14" nodemarkup="true"/>
      <nodename posx="1025" posy="689.5" maxwidth="200">
        <text data="Main Idea"/>
      </nodename>
      <nodenote>some note</nodenote>
      <nodes>
        <node id="1" posx="1242" posy="675.5" width="62" height="54" side="right" fold="false" treesize="54" color="#c6262e" colorroot="false" layout="Horizontal" group="false">
          <style branchmargin="100" branchradius="25" linktype="curved" linkwidth="4" linkarrow="false" linkdash="solid" nodeborder="rounded" nodewidth="200" nodeborderwidth="4" nodefill="true" nodemargin="8" nodepadding="6" nodefont="Sans 14" nodemarkup="true"/>
          <nodename posx="1256" posy="689.5" maxwidth="200">
            <text data="test"/>
          </nodename>
          <nodenote></nodenote>
        </node>
      </nodes>
    </node>
  </nodes>
  <groups/>
  <stickers/>
</minder>
```

**Comment by phase1geo** [comment]
With the latest code in the master branch, I am unable to replicate this issue.

**Comment by phase1geo** [comment]
I believe that this issue is now fixed in the master branch and will be generally available in the 1.15.6 release.
